app/policies/application_policy.rb

```diff
@@ -0,0 +1,53 @@
+class ApplicationPolicy
+  attr_reader :user, :record
+
+  def initialize(user, record)
+    @user = user
+    @record = record
+  end
+
+  # def index?
```

Comment:
You generally want to leave these methods uncommented. Basically, they're here to by default deny access, so that in the policies for specific models you override the ones you want to allow access to. This means that if don't explicitly allow access to some action then it won't be allowed. That way if you forget to set up part of a policy it won't expose information or allow access by accident.